packages/docs/content/components/form-label.mdx

```diff
@@ -34,37 +34,49 @@ The following Sass variables can be overridden to customize Form components:
 
 ## Guidance
 
-### When to use
-
-#### Labels
-
-- Each form field should have a `<label>`. Never use a field's placeholder attribute as the primary way to label the field.
-
-#### Legends
+### Form legend
 
 - Use a single legend for fieldset (this is required). One example of a common use of fieldset and legend is a question with radio button options for answers. The question text and radio buttons are wrapped in a fieldset, with the question itself being inside the legend tag.
+- Avoid links in legends when possible. Links inside legends are not read by some screen readers, most notably Internet Explorer 11, paired with JAWS. If you need a pattern like this, put the link outside of the legend element, but keep all other useful hint text in the legend.
 
-### Usage
 
-#### Labels
+### Form label
 
+- Each form field should have a `<label>`. Never use a field's placeholder attribute as the primary way to label the field.
 - Each form field should have a clear, brief, and descriptive label.
 - Labels should have a `for` attribute, referencing the corresponding input's unique `id` attribute. Only one label can be associated to each unique form element.
 - Labels should be placed above their fields.
 - Label text should be short and in sentence case.
 - Avoid colons at the end of labels.
 - Don't use a field's placeholder attribute as the primary way to label the field.
 
-#### Legends
+#### Content
 
-- Avoid links in legends when possible. Links inside legends are not read by some screen readers, most notably Internet Explorer 11, paired with JAWS. If you need a pattern like this, put the link outside of the legend element, but keep all other useful hint text in the legend.
+- Describe what the user needs to do when the field is selected
+- Use a phrase or complete sentence
+- Start with an action verb and pair it with a noun to help it be clearer. Example: Enter your age.
+- Use the least amount of words possible (i.e. don't use "please")
```

Comment:
```suggestion
- Use the fewest words possible (i.e. don't use "please")
```

Reply:
Does this bullet point contradict the one 2 points before (use a phrase or complete sentence)?

Do we care if folks to use numerically fewer words, or are we asking them to use direct/active language when writing labels?

Also, the "i.e." here should probably be "e.g." as "please" is being used as an example.